openslides/assignments/models.py

```diff
@@ -241,12 +241,12 @@ def create_poll(self):
         candidates = self.candidates.all()
 
         # Find out the method of the election
-        if config['assignment_poll_vote_values'] == 'votes':
+        if config['assignments_poll_vote_values'] == 'votes':
             yesnoabstain = False
-        elif config['assignment_poll_vote_values'] == 'yesnoabstain':
+        elif config['assignments_poll_vote_values'] == 'yesnoabstain':
             yesnoabstain = True
         else:
-            # config['assignment_poll_vote_values'] == 'auto'
+            # config['assignments_poll_vote_values'] == 'auto'
```

Comment:
This line is for readability so you can easily see that there are 3 cases and which cases there are.